ovs/dal/hybrids/vdisk.py

```diff
@@ -296,3 +300,15 @@ def calculate_delta(key, dynamic, current_stats):
                 else:
                     current_stats['{0}_ps'.format(key)] = max(0, (current_stats[key] - previous_stats[key]) / delta)
         volatile.set(prev_key, current_stats, dynamic.timeout * 10)
+
+    def reload_clients(self, client):
+        """
+        Reloads the StorageDriverClient and ObjectRegistryClient
+        """
+        if self.vpool_guid:
+            self._frozen = False
```

Comment:
I don't understand this ```self._frozen``` logic here. If it acts like a lock, shouldn't it be the other way around then?
```frozen = True``` before re-establishing the client and ```frozen = False``` afterwards